ui/client/package.json

```diff
@@ -6,49 +6,49 @@
   "license": "UNLICENSED",
   "private": true,
   "scripts": {
-    "build": "NODE_ENV=production webpack -p --progress && cp -R assets/icons/license dist",
+    "build": "NODE_ENV=production webpack --progress && cp -R assets/icons/license dist",
     "check-types": "tsc",
     "lint": "eslint --ext .ts,.tsx,.js,.jsx",
     "svgo": "svgo --config .svgo.yml",
     "check": "npm run check-types && npm run lint -- . --rule 'i18next/no-literal-string: warn'",
-    "start": "BACKEND_DOMAIN=http://localhost:8080 webpack-dev-server",
-    "start-backend-demo": "BACKEND_DOMAIN=https://staging.nussknacker.io webpack-dev-server",
+    "start": "BACKEND_DOMAIN=http://localhost:8080 webpack serve",
+    "start-backend-demo": "BACKEND_DOMAIN=https://staging.nussknacker.io webpack serve",
     "pretest": "npm run check",
-    "test": "./node_modules/karma/bin/karma start --single-run"
+    "test": "jest"
```

Comment:
Please update `README.md` in `client` dir. After this change, no configuration in Idea is needed - everything works just fine from scratch.